packages/elements/package.json

```diff
@@ -24,28 +24,39 @@
     "@storybook/addons": "^5.3.18",
     "@storybook/preset-typescript": "^3.0.0",
     "@storybook/react": "^5.3.18",
+    "@types/enzyme": "3.10.5",
+    "@types/enzyme-adapter-react-16": "1.0.6",
     "@types/jest": "^25.1.5",
     "@types/react": "^16.9.32",
+    "@types/react-dom": "16.9.6",
     "@types/react-test-renderer": "^16.9.2",
+    "@types/styled-components": "5.0.1",
     "babel-loader": "^8.1.0",
-    "babel-preset-react-app": "^9.1.2",
-    "identity-obj-proxy": "^3.0.0",
+    "babel-preset-react-app": "9.1.2",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
     "jest": "^25.2.6",
+    "jest-environment-enzyme": "7.1.2",
+    "jest-enzyme": "7.1.2",
+    "jest-transform-stub": "2.0.0",
     "prettier": "^2.0.2",
     "react-test-renderer": "^16.13.1",
```

Comment:
Done, thanks.

Reply:
When removing modules, please use `npm rm` otherwise the `package.json` file and the `package-lock.json` file will be out of sync. You can also do some changes to the `'package.json` file manually and then run `npm install` to sync the lock file.

Reply:
Yep this is how I usually manage packages (although I use the alias `uninstall`). I ran `install` but there were no changes on the lock file. I did remove `@types/react-test-renderer` though as it was no longer used. Is there something on the lock file that shouldn't be there?

Reply:
If it's up to date, then it's fine. I saw this commit https://github.com/creately/modules-js/pull/22/commits/8f2d10e700dcf80149be1a43bcdf1d286f1cb4fc which has a package.json change butt no changes on package-lock.json.

Reply:
Hmm yes that's strange...